Pages/js/options.js

```diff
@@ -411,7 +411,7 @@ function removeUrlInputs (event) {
   if(target.className.indexOf('remove_whitelist') >= 0) {
 
     var domainToRemove = target.getAttribute("data-value-to-remove");
-    var CSVToRemove = target.getAttribute("data-value-to-remove");
+    //var CSVToRemove = target.getAttribute("data-value-to-remove");
```

Comment:
yeah, why not.

2015-03-25 0:08 GMT+02:00 Sean McGregor <notifications@github.com>:

> In Pages/js/options.js
> <https://github.com/privly/privly-applications/pull/226#discussion_r27078671>
> :
>
> > @@ -411,7 +411,7 @@ function removeUrlInputs (event) {
> >    if(target.className.indexOf('remove_whitelist') >= 0) {
> >
> >      var domainToRemove = target.getAttribute("data-value-to-remove");
> > -    var CSVToRemove = target.getAttribute("data-value-to-remove");
> > +    //var CSVToRemove = target.getAttribute("data-value-to-remove");
>
> If it isn't used, you should just remove it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/privly/privly-applications/pull/226/files#r27078671>.
>